scripts/kani-regression.sh

```diff
@@ -41,13 +41,9 @@ cargo test -p kani-compiler
 cargo test -p kani-driver
 cargo test -p kani_metadata
 
-# Check output files (--gen-c option)
-echo "Check GotoC output file generation"
-time "$KANI_DIR"/tests/output-files/check-output.sh
-echo ""
-
 # Declare testing suite information (suite and mode)
 TESTS=(
+    "exec-init exec"
```

Comment:
Are we planning to add the other script-based scripts to the same suite? If so, maybe rename `exec-init` to `script-based` or something more generic?

Reply:
Renamed it to `script-based-init` - the `init` is supposed to indicate that these are the initial tests, because I intend to migrate the script-based tests that occur in the end (e.g., `std` test) to another suite `script-based-end`. How does that sound?

Reply:
Nit: In this case "script-based-pre" might be a bit clearer.

Reply:
Sure, done.